code-challenges.rb

```diff
@@ -31,3 +41,13 @@
 # Expected output: 'LEARN is not a palindrome'
 is_palindrome3 = 'Rotator'
 # Expected output: 'Rotator is a palindrome'
+
+def palindrome? (str)
+    if str.downcase == str.downcase.reverse
+       p "#{str} is a palindrome."
+    else
+       p "#{str} is not a palindrome."
+    end
+end
+
+palindrome? is_palindrome3
```

Comment:
I would caution you against using questions marks in your custom Ruby methods. That has a specific meaning. Also, remember to use the `p/puts` on the method invocation and not in the inner workings of your method.